### ISSUE TITLE
[Bug]: Move "bindsUserNewsResourceRepository" to "DataModule"

### ISSUE THREAD

**Issue by lelelongwang** [android/nowinandroid#903]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is there a StackOverflow question about this issue?

- [X] I have searched StackOverflow

### What happened?

There is already interface `DataModule`, should abstract method `bindsUserNewsResourceRepository` be written in interface `DataModule`?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SimonMarquis** [comment]
@lelelongwang I agree with you. `DataModule` already binds all repositories of the `:core:data` module, except for `UserNewsResourceRepository`.
You can create a PR for that if you want.

**Comment by lelelongwang** [comment]
Ok, I'll deal with it soon, thanks for the reply

**Comment by Code-indomitus** [comment]
Hi, I am looking to work on open-source projects. I have just started learning Android and jetpack compose. How can I start contributing to issues over here in this repo? Any guidance would be much appreciated.
